storage_backend_s3/tests/test_fake_s3.py

```diff
@@ -0,0 +1,44 @@
+# Copyright 2019 KMEE (http://www.akretion.com).
+# @author Luis Felipe Mileo <mileo@kmee.com.br>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+import logging
+import os
+from odoo.addons.storage_backend.tests.common import Common
+
+_logger = logging.getLogger(__name__)
+
+
+class FakeS3Case(Common):
```

Comment:
any reason not to update the existing one?

https://github.com/OCA/storage/blob/5c423809e64007007f5a21d06bc64974f96923f7/storage_backend_s3/tests/test_amazon_s3.py#L46